app/models/card.rb

```diff
@@ -37,4 +43,28 @@ def text_are_not_equal
       errors.add(:translated_text, 'Текст перевода не должен быть равен оригинальному тексту')
     end
   end
-end
+
+
+  def date_for_review(attempt)
+    case attempt
+    when 1 then
+    update_review_date((DateTime.current+12.hour).strftime('%H'))
+    when 2 then
+    update_review_date((DateTime.current+3.day).to_date)
```

Comment:
Use 2 (not 0) spaces for indentation.<br>Surrounding space missing for operator '+'.